packages/block-library/src/categories/block.json

```
@@ -26,6 +26,10 @@
 		"showEmpty": {
 			"type": "boolean",
 			"default": false
+		},
+		"taxonomy": {
+			"type": "string",
+			"default": 'category'
```

```suggestion
			"default": "category"
```

It seems to work if this is fixed.